sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -9,13 +9,13 @@ EGIT_REPO_URI="http://llvm.org/git/libcxx.git
 	https://github.com/llvm-mirror/libcxx.git"
 CMAKE_MIN_VERSION=3.4.3
 
-[ "${PV%9999}" != "${PV}" ] && SCM="git-r3" || SCM=""
+[[ ${PV} == 9999 ]] && SCM="git-r3" || SCM=""
```

Comment:
[[ ${PV} == *9999 ]]

means **exactly** the same, is more readable and less likely to confuse people on what exactly it's supposed to mean.

Even if we were limited to POSIX shell (which we are not), you wouldn't have a reason to use such an ugly syntax.